src/job/job.resolver.ts

```diff
@@ -39,4 +41,9 @@ export class JobResolver {
   async workflows(@Parent() job: Job): Promise<Workflow[]> {
     return this.workflowService.findByIds(job.workflows.map((workflow) => workflow._id));
   }
+
+  @ResolveField(() => Comment)
```

Comment:
Just a small fix, should specify the type as `[Comment]` so that GraphQL knows the return type is a list of comments instead of a single comment.

```suggestion
  @ResolveField(() => [Comment])
```